src/lib/create-config.ts

```diff
@@ -333,11 +333,8 @@ const defaultAuthentication: IAuthOption = {
 
 const defaultImport: WithOptional<IImportOption, 'file'> = {
     file: process.env.IMPORT_FILE,
-    dropBeforeImport: parseEnvVarBoolean(
-        process.env.IMPORT_DROP_BEFORE_IMPORT,
-        false,
-    ),
-    keepExisting: parseEnvVarBoolean(process.env.IMPORT_KEEP_EXISTING, false),
+    project: process.env.IMPORT_PROJECT ?? 'default',
+    environment: process.env.IMPORT_ENVIRONMENT ?? 'development',
```

Comment:
we can ask our users what env they use in the old import most often

Reply:
I've reached out to the user requesting the feature here:
https://github.com/Unleash/unleash/issues/6306#issuecomment-2141404520